README.md

```diff
@@ -49,6 +51,26 @@ the image first.
 
 - When confirmed, click on the `Flash` button to start flashing procedure.
 
+### bmaptool
+
+If you prefer a command-line tool, then you should use bmaptool, just install it with
+
+```
+sudo apt install bmap-tools
+```
+
+then run with
+
+```
+sudo bmaptool copy path/to/image/example.wic.gz /dev/yourSDcard
+```
+
+remember, the images are located in the build directory, for example like this:
```

Comment:
@WiktorG351 as I said above, let's use images from https://github.com/3mdeb/meta-rte/releases

Reply:
fixed in https://github.com/3mdeb/meta-rte/blob/734d074babc8652f7e6ba7f7ada1e5a296b60f0b/README.md?plain=1#L73